build-tools/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -1209,8 +1209,7 @@ private void sync(Path sourceRoot, Path destinationRoot, BiConsumer<Path, Path>
                 }
             });
         } catch (UncheckedIOException e) {
-            if (e.getCause() instanceof NoSuchFileException) {
-                NoSuchFileException cause = (NoSuchFileException) e.getCause();
+            if (e.getCause()instanceof NoSuchFileException cause) {
```

Comment:
nit: missing space

```suggestion
            if (e.getCause() instanceof NoSuchFileException cause) {
```

Reply:
@henningandersen unfortunately, that whitespace missing there is the result of spotless, because of a bug. https://bugs.eclipse.org/bugs/show_bug.cgi?id=574437
Even if it was fixed on the Eclipse side, we need to wait for Spotless to upgrade. I _think_ https://github.com/diffplug/spotless/issues/1046 this is the issue we are waiting for to be fixed.